api/v1alpha1/frontend_types.go

```diff
@@ -199,7 +199,8 @@ type Route struct {
 }
 
 type Analytics struct {
-	APIKey string `json:"APIKey" yaml:"APIKey"`
+	APIKey    string `json:"APIKey" yaml:"APIKey"`
+	APIKeyDev string `json:"APIKeyDev" yaml:"APIKeyDev"`
```

Comment:
One thing I've noticed. We should add the `omitempty` flag here. If we don't it may cause existing configurations to fail because it will be missing.